packages/Webkul/BookingProduct/src/Resources/lang/ben/app.php

```diff
@@ -116,7 +116,7 @@
             'special-notes'            => 'বিশেষ অনুরোধ/নোট',
             'event-on'                 => 'ইভেন্ট চালু',
             'book-your-ticket'         => 'আপনার টিকিট বুক করুন',
-            'per-ticket-price'         => ':price টিকিট প্রতি',
+            'per-ticket-price'         => 'প্রতি টিকিটের মূল্য',
```

Comment:
Hi,

You translated :price variable also, which will not provide the correct string.